netlify/edge-functions/components/Pill.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
I'll probably remove this as importing it doesn't seem to work in an edge function. Still testing that theory out.